org-brain.el

```diff
@@ -62,24 +64,94 @@ This will be used by `org-brain-new-child'."
   :group 'org-brain
   :type '(string))
 
+;;; Utils
+(defun flatten (obj)
+  (do* ((result (list obj))
+        (node result))
+       ((null node) (delete nil result))
+    (cond ((consp (car node))
+           (when (cdar node) (push (cdar node) (cdr node)))
+           (setf (car node) (caar node)))
+          (t (setf node (cdr node))))))
+
+;;; Logging
+(defcustom org-brain-log t
+  "Set to nil to not write to *Messages* buffer."
+  :group 'org-brain
+  :type 'boolean)
+
+(defun org-brain-log (msg)
+  (when org-brain-log
+    (setq inhibit-message t)
+    (message msg)
+    (setq inhibit-message nil)))
+
+;;; Caches
+(setq org-brain-files-cache nil)
+(setq org-brain-parents-cache nil)
+(setq org-brain-children-cache nil)
+(setq org-brain-pins-cache nil)
+
+(defun org-brain-invalidate-all-caches ()
```

Comment:
It's just a comment before the function definition (see some of the other autload cookies in the code, like `org-brain-visualize`).